### PR TITLE
Specify randomized responses for navigation sources

### PR DESCRIPTION
**Pull request by apasel422**
#386

***
<a href="https://pr-preview.s3.amazonaws.com/apasel422/conversion-measurement-api/pull/419.html" title="Last updated on May 11, 2022, 12:28 PM UTC (3e0dc69)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/419/f56b39e...apasel422:3e0dc69.html" title="Last updated on May 11, 2022, 12:28 PM UTC (3e0dc69)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -699,32 +760,12 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
...
-        : [=attribution trigger/priority=]
-        :: 0
-        : [=attribution trigger/filters=]
-        :: «[]»
-        : [=attribution trigger/debug key=]
-        :: null
-    1. Let |fakeReport| be the result of running [=obtain a report=] with |source| and |fakeTrigger|.
-    1. [=set/Append=] |fakeReport| to the [=attribution report cache=].
-    1. Return.
+1. If |source|'s [=attribution source/randomized response=] is a [=list=]:
```
Comment (csharrison) [inline index.bs:763]: "is non-null" maybe? Otherwise isn't it more precise to say "is a [=set=]?

**Review by csharrison — COMMENTED** [review]
Well done. LGTM!

**Review by csharrison — APPROVED** [review]

**Review by johnivdel — COMMENTED** [review]
(a few questions on this, acknowledging its already merged)

File: index.bs [hunk 2]
```diff
@@ -1013,34 +1054,57 @@ can be associated with a source event id. Larger values allow for more attributi
...
+
+1. If |sourceType| is "`event`", return «».
+1. Return « (2 days - 1 hour), (7 days - 1 hour) ».
+
+To <dfn>obtain the number of report windows</dfn> given a [=source type=]
+|sourceType|:
+
+1. Let |earlyDeadlines| be the result of running [=obtain early deadlines=]
+    with |sourceType|.
+1. Return the [=list/size=] of |earlyDeadlines| + 1.
```
[the remark this answers is not in the document]
Comment (apasel422) [reply index.bs:1073]: So is this a problem with [the implementation](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_utils.cc;drc=8c8c71848440706a0e534c9a5a91a1792d06a894) as well?

[the remark this answers is not in the document]
Comment (csharrison) [reply index.bs:1073]: Ah yes I think this is a real problem. Man, I liked the solution here... :(

**Pull request merged**